test/development/app-dir/capture-console-error-owner-stack/capture-console-error-owner-stack.test.ts

```diff
@@ -201,8 +201,57 @@ describe('app-dir - capture-console-error-owner-stack', () => {
 
     const result = await getRedboxResult(browser)
 
-    if (process.env.TURBOPACK) {
-      expect(result).toMatchInlineSnapshot(`
+    if (process.env.__NEXT_EXPERIMENTAL_NEW_DEV_OVERLAY === 'true') {
+      if (process.env.TURBOPACK) {
+        expect(result).toMatchInlineSnapshot(`
+         {
+           "callStacks": "Page
+         app/rsc/page.js (2:17)
+         JSON.parse
+         <anonymous> (0:0)
+         Page
+         <anonymous> (0:0)",
+           "count": 1,
+           "description": "Error: boom",
+           "source": "app/rsc/page.js (2:17) @ Page
+
+           1 | export default function Page() {
+         > 2 |   console.error(new Error('boom'))
+             |                 ^
+           3 |   return <p>rsc</p>
+           4 | }
+           5 |",
+           "title": "Console Error
+         Server",
```

Comment:
This badge need to be a snapshot in the redbox result, we can't mix it into the title property, it's misleading that "Server" is part of the title. We need to create a new util  to locate the badge and read the text 
```suggestion
         Server",
```

Reply:
The `title` became the badges in the new overlay. I agree that we need to change related attributes and naming.
Can we do that in a follow-up?

Reply:
Can stay for now until I finish the `toDisplayRedbox` matcher.

Reply:
Will be a different field in the new Redbox snapshot matchers: https://github.com/vercel/next.js/pull/73621